src/main/kotlin/no/skatteetaten/aurora/cantus/controller/DockerRegistryController.kt

```diff
@@ -0,0 +1,41 @@
+package no.skatteetaten.aurora.cantus.controller
+
+import no.skatteetaten.aurora.cantus.service.DockerRegistryService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class DockerRegistryController(val dockerRegistryService: DockerRegistryService) {
+
+    @GetMapping("/affiliation/{affiliation}/name/{name}/tag/{tag}/manifest")
```

Comment:
Dersom man ønsker bruke pathvariable her bør ikke leddene kvalifiseres med navn. Bruk isteden: "/{affiliation}/{name}/{tag}/manifest"

Reply:
Ser bedre ut. Var usikker på hvilken av de to som gir mest mening.